CMakeLists.txt

```diff
@@ -1,21 +1,30 @@
-set(_ct_min_cmake_version "3.14" CACHE INTERNAL "")
-cmake_minimum_required(VERSION "${_ct_min_cmake_version}") #Required for FetchContent_MakeAvailable()
+#3.19 needed because of CMaize
+set(_ct_min_cmake_version "3.19" CACHE INTERNAL "")
+cmake_minimum_required(VERSION "${_ct_min_cmake_version}")
 project(CMakeTest VERSION 1.0.0 LANGUAGES NONE)
 
 if(NOT CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
-	set(_ct_min_cmake_version "${_ct_min_cmake_version}" PARENT_SCOPE) #Make sure projects using CMakeTest have this variable
+    #Make sure projects using CMakeTest have this variable
+	set(_ct_min_cmake_version "${_ct_min_cmake_version}" PARENT_SCOPE)
 endif()
 
+# Add cmake directory to CMake's module path, iff it's not already there
 list(FIND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" path_has_cmake)
 if(${path_has_cmake} STREQUAL "-1")
     list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 endif()
+
+# TODO: Should this be in the above if block?
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" CACHE STRING "" FORCE)
```

Comment:
I don't think it hurts for this to be outside of the above if block. At the worst, I think it is just writing the same `CMAKE_MODULE_PATH` that is already in the cache when the if statement is not triggered. You could probably save a few cycles if it was in the if statement, but not enough to really matter.